internal/app/subsystems/aio/store/postgres/postgres.go

```diff
@@ -470,11 +370,7 @@ func (s *PostgresStore) String() string {
 }
 
 func (s *PostgresStore) Start() error {
-	if _, err := s.db.Exec(CREATE_TABLE_STATEMENT); err != nil {
-		return err
-	}
-
-	return nil
+	return store.Start(Version, s.db, 10*time.Second, migrationsFS, s.config.Plan)
```

Comment:
Could we rename this function to something to indicate this is a migration? Another possibility would be to add a new function called `Migrate` and have start call that function, then the migrate command could also call that function directly and avoid the start/stop.